samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/pipeline.py

```diff
@@ -0,0 +1,41 @@
+import kfp.dsl as dsl
+import kfp.components as components
+from kfp.gcp import use_gcp_secret
+
+@dsl.pipeline(
+    name = "kaggle pipeline",
+    description = "kaggle pipeline that go from download data, train model to display result"
+)
+def kaggle_houseprice(
+    bucket_name: str,
+    commit_sha: str
+):
+    import os
+
+    downloadDataOp = components.load_component_from_file('./download_dataset/component.yaml')
+    downloadDataStep = downloadDataOp(bucket_name=bucket_name).apply(use_gcp_secret('user-gcp-sa'))
```

Comment:
Similar comment as in another PR: perhaps we'll need to explicitly document the usage of GCP SA here. Especially given that in latest releases workload-identity became the preferred approach.

Reply:
Ok. Will add illustrations in readme.md.